daemon/inertiad/auth/users.go

```diff
@@ -230,9 +230,9 @@ func (m *userManager) IsAdmin(username string) (bool, error) {
 	return admin, err
 }
 
-// isTOTPEnabled checks if a given user has TOTP enabled
-func (m *userManager) IsTOTPEnabled(username string) (bool, error) {
-	TOTPenabled := false
+// isTotpEnabled checks if a given user has Totp enabled
+func (m *userManager) IsTotpEnabled(username string) (bool, error) {
+	Totpenabled := false
```

Comment:
`totpEnabled` - variables should be lowercased, since in Go capitalised variables have special meanings